vocode/streaming/streaming_conversation.py

```diff
@@ -1004,15 +1004,16 @@ async def send_initial_message(self, initial_message: BaseMessage):
         )
         # TODO: configure if initial message is interruptible
         initial_message_tracker = asyncio.Event()
-        agent_response_event = (
-            self.interruptible_event_factory.create_interruptible_agent_response_event(
-                AgentResponseMessage(message=initial_message),
-                is_interruptible=False,
-                agent_response_tracker=initial_message_tracker,
+        # If it is outbound, update the history with the initial message
+        if self.agent.get_agent_config().call_type == CallType.OUTBOUND:
+            self.agent.update_history(
+                "human", self.transcriptions_worker.buffer.to_message()
```

Comment:
we don't want to use the tracker for outbound?

Reply:
Ummm this is the human message.

Reply:
Outbound we add the current transcription. Then both inbound and outbound update the history on line 1012 for the bot.